model/src/main/java/org/cbioportal/genome_nexus/model/Version.java

```diff
@@ -0,0 +1,28 @@
+package org.cbioportal.genome_nexus.model;
+
+public class Version {
+
+    private String version;
+    private boolean isStatic;
+
+    public Version(String version, boolean isStatic) {
+        this.version = version;
+        this.isStatic = isStatic;
+    }
+
+    public String getVersion() {
+        return version;
+    }
+
+    public void setVersion(String version) {
+        this.version = version;
+    }
+
+    public boolean isStatic() {
```

Comment:
I think somewhere we should have a description (to the end user) of what Static means. There is not really a good single word which captures what we mean by this boolean ... something like "expected to deliver unchanging results for a given request until the string accessed through getVersion() changes"